packages/pluggableWidgets/image-viewer-web/src/ImageViewer.tsx

```diff
@@ -46,9 +46,19 @@ function getImageProps({
     }
 }
 
+function processImageLink(imageLink: string | undefined, displayAs: DisplayAsEnum): string | undefined {
+    if (!imageLink || displayAs === "fullImage") {
+        return imageLink;
+    }
+    const url = new URL(imageLink);
+    url.searchParams.append("thumb", "true");
+    return url.href;
+}
+
 export const ImageViewer: FunctionComponent<ImageViewerContainerProps> = props => {
     const onClick = useCallback(() => props.onClick?.execute(), [props.onClick]);
     const { type, image } = getImageProps(props);
+    const processedImageLink = type === "image" ? processImageLink(image, props.displayAs) : image;
```

Comment:
We should use the thumb only if we don't have an expand on click, otherwise we will never have the original image being shown even in the modal.